packages/@okta/vuepress-site/docs/reference/api/schemas/index.md

```diff
@@ -1037,6 +1037,504 @@ curl -v -X POST \
+| definitions | Subschema with one or more custom Profile properties | Body       | [Group Profile custom subschema](#group-profile-custom-subschema) | TRUE     |
+
+##### Response parameters
+
+
+[Group Schema](#Group-schema-object)
+
+##### Request example
+
+
+```bash
+curl -v -X POST \
+-H "Accept: application/json" \
+-H "Content-Type: application/json" \
+-H "Authorization: SSWS ${api_token}" \
+-d '{
+  "definitions": {
+    "custom": {
+      "id": "#custom",
+      "type": "object",
+      "properties": {
+        "groupContact": {
+          "title": "Group administrative contact",
+          "description": "Group administrative contact",
+          "type": "string",
+          "required": false,
+          "minLength": 1,
+          "maxLength": 20,
+          "permissions": [
+            {
+              "principal": "SELF",
+              "action": "READ_WRITE"
+            }
+          ]
+        }
+      },
+      "required": []
+    }
+  }
+}' "https://${yourOktaDomain}/api/v1/meta/schemas/group/default"
+```
+
+##### Response example
+
+The following response is only a subset of properties for brevity.
+
+```json
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "_links": {
+    "self": {
+      "href": "http://${yourOktaDomain}/api/v1/meta/schemas/group/default",
+      "method": "GET",
+      "rel": "self"
+    }
+  },
+  "created": "2021-01-30T00:18:24.000Z",
+  "definitions": {
+    "base": {
+      "id": "#base",
+      "properties": {},
+      "required": [
+        "name"
+      ],
+      "type": "object"
+    },
+    "custom": {
+      "id": "#custom",
+      "properties": {
+        "groupContact": {
+          "description": "Group administrative contact",
+          "master": {
+            "type": "PROFILE_MASTER"
+          },
+          "mutability": "READ_WRITE",
+          "permissions": [
+            {
+              "action": "READ_WRITE",
+              "principal": "SELF"
+            }
+          ],
+          "scope": "NONE",
+          "title": "Group administrative contact",
+          "type": "string"
+        }
+      },
+      "required": [],
+      "type": "object"
+    }
+  },
+  "description": "Okta group profile template",
+  "id": "https://${yourOktaDomain}/meta/schemas/group/default",
+  "lastUpdated": "2021-02-25T23:05:31.000Z",
+  "name": "group",
+  "properties": {
+    "profile": {
+      "allOf": [
+        {
+          "$ref": "#/definitions/custom"
+        },
+        {
+          "$ref": "#/definitions/base"
+        }
+      ]
+    }
+  },
+  "title": "Okta group",
+  "type": "object"
+}
+```
+
+### Remove property from Group Profile Schema
+
+<ApiOperation method="post" url="/api/v1/meta/schemas/group/default" />
+
+Removes one or more [custom Group Profile properties](#group-profile-schema-property-object) from the group schema.  Currently Okta does not support removing [base Group Profile properties](#group-profile-base-subschema).
```

Comment:
Remove extra space after period.

Comma after "Currently".

does not > doesn't